packages/messaging/src/controllers/sw-controller.ts

```diff
@@ -297,7 +297,7 @@ function hasVisibleClients(clientList: WindowClient[]): boolean {
       client.visibilityState === 'visible' &&
       // Ignore chrome-extension clients as that matches the background pages
       // of extensions, which are always considered visible for some reason.
-      !client.url.startsWith('chrome-extension://')
+      !(client.url.startsWith('chrome-extension://') && client.url.endsWith("_generated_background_page.html"))
```

Comment:
Looks like this will work for auto-gen background.html. The next step is probably to have a more generic solution that would work for all situation (user defined background html) and across browser if applicable 😎